tensorrt_llm/_torch/modules/fused_moe/mega_moe/mega_moe_deepgemm.py

```
@@ -359,6 +371,88 @@ def validate_configurable_moe(self, moe) -> None:
+        except ImportError:
+            return
+        comm = MPI.COMM_WORLD
+        rank = comm.Get_rank()
+        world_size = comm.Get_size()
+        if world_size <= 1:
+            return
+
+        try:
+            local_comm = comm.Split_type(MPI.COMM_TYPE_SHARED)
+            local_rank = local_comm.Get_rank()
+        except Exception:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)